api/verify_test.go

```diff
@@ -129,45 +123,38 @@ func (ts *VerifyTestSuite) TestVerifySecureEmailChange() {
 	assert.False(ts.T(), u.IsConfirmed())
 
 	// Verify new email
-	var vbuffer bytes.Buffer
-	require.NoError(ts.T(), json.NewEncoder(&vbuffer).Encode(map[string]interface{}{
-		"type":  "email_change",
-		"token": u.EmailChangeTokenNew,
-	}))
-
-	req = httptest.NewRequest(http.MethodPost, "http://localhost/verify", &vbuffer)
-	req.Header.Set("Content-Type", "application/json")
+	reqURL := fmt.Sprintf("http://localhost/verify?type=%s&token=%s", emailChangeVerification, u.EmailChangeTokenNew)
+	req = httptest.NewRequest(http.MethodGet, reqURL, nil)
 
 	w = httptest.NewRecorder()
 	ts.API.handler.ServeHTTP(w, req)
 
-	data := make(map[string]interface{})
-	require.Equal(ts.T(), http.StatusOK, w.Code)
-	require.NoError(ts.T(), json.NewDecoder(w.Body).Decode(&data))
-	require.Equal(ts.T(), singleConfirmationAccepted, data["msg"])
+	require.Equal(ts.T(), http.StatusSeeOther, w.Code)
+	urlVal, err := url.Parse(w.Result().Header.Get("Location"))
```

Comment:
Didn't know about location prior -- for anyone else reading [here's the docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location)